Sources/Winget-Install-GUI.ps1

```diff
@@ -680,8 +791,46 @@ $Script:stream = [System.IO.MemoryStream]::new($IconBase64, 0, $IconBase64.Lengt
 #Check if WiGui is uptodate
 Get-WiGuiLatestVersion
 
-#Check if Winget is installed, and install if not
-Get-WingetStatus
+#Check if Winget is installed, and install if not (and download favourite apps/set ACL if in WSB)
+$Script:WhoAmI = & whoami

```

Comment:
I think native powershell environnment variable is:  
`$env:UserName`

Reply:
I'm a dinosaur🤪